File: src/main/java/com/wyan/service/AdSpaceService.java
package com.wyan.service;

import com.github.pagehelper.PageInfo;
import com.wyan.pojo.TbAdSpace;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface AdSpaceService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbAdSpace> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbAdSpace ad_space);


	/**
	 * 修改
	 */
	public void update(TbAdSpace ad_space);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbAdSpace findOne(Integer spaceid);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] spaceids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbAdSpace ad_space, int pageNum, int pageSize);
	
}

File: src/main/webapp/js/service/AdSpaceService.js
//服务层
app.service('adSpaceService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../adSpace/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../adSpace/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../adSpace/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../adSpace/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../adSpace/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../adSpace/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../adSpace/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});